src/patternfly/demos/MasterDetail/master-detail-panel-header.hbs

```diff
@@ -0,0 +1,15 @@
+{{#> drawer-head}}
+  {{#> drawer-actions}}
```

Comment:
should you wrap this in `{{#unless drawer--IsStatic}}{{/unless}}` so it isn't in static drawers?

Reply:
No, the responsive layout needs requires a close button.